src/cascadia/TerminalSettingsModel/AIConfig.cpp

```diff
@@ -18,7 +18,48 @@ static constexpr std::wstring_view PasswordVaultAIKey = L"TerminalAIKey";
 static constexpr std::wstring_view PasswordVaultAIEndpoint = L"TerminalAIEndpoint";
 static constexpr std::wstring_view PasswordVaultOpenAIKey = L"TerminalOpenAIKey";
 static constexpr std::wstring_view PasswordVaultGithubCopilotAuthToken = L"TerminalGithubCopilotAuthToken";
-static constexpr std::wstring_view PasswordVaultGithubCopilotRefreshToken = L"TerminalGithubCopilotRefreshToken";
+static constexpr std::wstring_view PasswordVaultGithubCopilotRefreshToken = L"TerminalGithubCopilotRefreshToken";
+static constexpr std::wstring_view AzureOpenAIRegistryKey = L"AzureOpenAI";
+static constexpr std::wstring_view OpenAIRegistryKey = L"OpenAI";
+static constexpr std::wstring_view GitHubCopilotRegistryKey = L"GitHubCopilot";
```

Comment:
"Registry key" is confusing in this case as it it not a key path and not a value name. Better is "PolicyAllowedId<ProviderName>" or something.